themes/vocabulary_theme/templates/blocks/hero.html

```diff
@@ -1,7 +1,7 @@
-<section class="hero">
+<section class="hero" role="region" aria-labelledby="hero-title">
```

Comment:
`sections` already have a semantic `role`, this isn't necessary. Additionally the `aria-labelledby` shouldn't be necessary either.